src/world.cpp

```diff
@@ -139,7 +92,18 @@ World::update_single_mesh(TerrainDim3 tile_sop) {
 
 void
 World::update_all_chunks_mesh() {
-    for (size_t i = 0; i < chunks_mesh_.size(); i++)
+    size_t num_chunks = terrain_main_.get_chunks().size();
+    if (chunks_mesh_.size() != num_chunks) {
```

Comment:
```suggestion
    if (chunks_mesh_.size() != num_chunks) [[unlikely]] {
```

Reply:
When this function is called this is true. For that reason I don't want to put likely or unlikely.

Reply:
so `chunks_mesh_.size() != num_chunks` when this is called?

So its `[[likely]]`?

Reply:
This is a general problem with optimization. When this is called it should be considered likely, but if it were to be called many times, and the optimization were to matter it should be flaged unlikely. Because of the contradicting points I don't want to put a flag until the code is expanded, and some analysis can be done.

Reply:
Ok